### DISCUSSION TITLE
Trying Chat example, no error, no connection.

### DISCUSSION THREAD

**Discussion by Ekkas** [serverpod/serverpod#693] [Q&A] [answered]

**Comment by Ekkas** [comment]
Hi all, 
I'm implementing the examples to 'find my feet' with Serverpod.
I managed to get to the point where I believe the chat should start, but just get a circular progress.
Server-side shows user authenticated, but then authenticatedUser: null :

METHOD CALL: serverpod_auth.email.authenticate duration: 33ms numQueries: 0 authenticatedUser: 3
METHOD CALL: channels.getChannels duration: 6ms numQueries: 0 authenticatedUser: null

 and no error on Flutter-side.

Where can I see why the chat does not want to connect/start?
Thank you
Ekkas

**Reply by yahu1031** [reply]
>but just get a circular progress.

Share us this logic.

**Comment by vlidholt** [comment, marked as answer]
Have you added the tables for the authentication module, the chat module, and for the example?

**Comment by Ekkas** [comment]
Thank you vlidholt,
I skipped a table.
It seems to be working now!

Thanks
Ekkas
